### ISSUE TITLE
adoptedStyleSheets does not applies on 'closed' shadowRoot

### ISSUE THREAD

**Issue by ar2r13** [calebdwilliams/construct-style-sheets#65]
**Describe the bug**
adoptedStyleSheets does not apply on `{ type : 'closed' }` shadowRoot

**To Reproduce**
```js
const style = (new CSSStyleSheet).replaceSync(`:host { background: red }`)

class WebComponent extends HTMLElement {
   constructor () {
      super().attachShadow({ mode: 'closed' }).adoptedStyleSheets = [style]
   }
}
```

**Desktop (please complete the following information):**
 - Browser: Safari

**Comment by adi518** [comment]
But that's the idea of a shadow DOM being closed.

**Comment by calebdwilliams** [comment]
I have mixed thoughts here. I _could_ capture the shadow root when it's created and save a reference for myself, but that feels antithetical to closed shadow roots. I might add this when I switch to TypeScript. I'll leave this open for now.

**Comment by calebdwilliams** [comment]
This should be fixed in version 2.4.16.
